infrastructure/main.tf

```diff
@@ -277,7 +303,7 @@ resource "null_resource" "update_cloudfront" {
   count = coalesce(var.cloudfront_id, false) != false ? 1 : 0
 
   provisioner "local-exec" {
-    command = "${path.module}/cf_update.sh ${var.cloudfront_id} workflows_api_origin \"${aws_apigatewayv2_api.workflows_http_api.api_endpoint}\""
+    command = "${path.module}/cf_update.sh ${var.cloudfront_id} workflows-api \"${aws_apigatewayv2_api.workflows_http_api.api_endpoint}\""
```

Comment:
This is a change we opted out of in https://github.com/NASA-IMPACT/veda-data-airflow/pull/167 so we should double check if we still want it

Reply:
ah i think this branch was branched off of that branch so this sneaked in here

Reply:
removed